neptune-python-utils/neptune_python_utils/endpoints.py

```diff
@@ -169,8 +170,8 @@ def get_headers():
         
 
 class Endpoints:
-    
-    def __init__(self, neptune_endpoint=None, neptune_port=None, region_name=None, credentials=None, role_arn=None, proxy_dns=None, proxy_port=8182, remove_host_header=False):
+
+    def __init__(self, neptune_endpoint=None, neptune_port=None, region_name=None, credentials=None, role_arn=None, proxy_dns=None, proxy_port=8182, remove_host_header=False, sts_endpoint=None):
```

Comment:
nit maybe the uri can be passed in as a default param

Reply:
> nit maybe the uri can be passed in as a default param

I wanted to do that but it looked a bit unclean. Also wanted to pick the uri from env variables as it is done for certain params in this code but that also felt like I was over-complicating the solution.